src/Sarif.Viewer.VisualStudio/Sarif/CodeFlowLocation.extensions.cs

```diff
@@ -17,13 +17,16 @@ public static CodeFlowLocationModel ToCodeFlowLocationModel(this CodeFlowLocatio
             {
                 if (codeFlowLocation.Location.PhysicalLocation != null)
```

Comment:
You can DRY out this code by having `CodeFlowLocationExtensions.ToCodeFlowLocationModel` just call `LocationExtensions.ToCodeFlowLocationModel`:
```
public static CodeFlowLocationModel ToCodeFlowLocationModel(this CodeFlowLocation codeFlowLocation)
{
    return codeFlowLocation?.Location != null
        ? codeFlowLocation.Location.ToCodeFlowLocationModel()
        : new CodeFlowLocationModel();
}
```

Reply:
Good spotting